administration/src/mui-modules/base/AlertBox.tsx

```diff
@@ -3,10 +3,18 @@ import React, { ReactElement } from 'react'
 import { useTranslation } from 'react-i18next'
 
 const CenteredAlert = styled(Alert)`
-  margin: auto;
+  margin: 5px;
   display: flex;
   justify-content: center;
   align-items: center;
+
+  @media (max-width: 899px) {
+    flex-direction: column;
+    & .MuiAlert-action {
+      align-self: start;
+      margin-left: 0;
+    }
+  }
```

Comment:
💭  Here a simple `sx` prop could be used, along with the MUI defined breakpoints and grid system:

```tsx
    <Alert
      sx={theme => ({
        margin: 1,
        justifyContent: 'center',
        alignItems: 'center',
        [theme.breakpoints.down('md')]: {
          flexDirection: 'column',
          alignSelf: 'flex-start',
          marginLeft: 0,
        },
      })}
      ...
      >
    </Alert>